fbw-a32nx/src/systems/fmgc/src/components/ReadySignal.ts

```diff
@@ -16,6 +17,7 @@ export class ReadySignal implements FmgcComponent {
             && SimVar.GetSimVarValue('L:A32NX_IS_READY', 'number') !== 1) {
             // set ready signal that JS code is initialized and flight is actually started
             // -> user pressed 'READY TO FLY' button
+            SimVar.SetSimVarValue('L:A32NX_AIRCRAFT_TYPE', 'enum', AircraftType.A320_251N);
```

Comment:
This should probably not be done here but in the extras-host or systems-host.
I personally think extras-host would be a better place.

Or alternatively this could be defined in the systems.cfg
```ini
[LocalVars]
LocalVar.1 = A32NX_DEVELOPER_STATE
LocalVarDefault.1 = 0
; Aircraft Type:  A320_251N=0,  A380_842=1
LocalVar.1 = A32NX_AIRCRAFT_TYPE
LocalVarDefault.1 = 0
```

(How I hate this "A32NX_" prefix - in this case it is so clear that it should have never been done like this.)

Reply:
Thx for the review and feedback, will make the changes asap.

I'm either not a fan of the prefix, but it is what it is - a refactor of this has a to much impact on external stuff (software, cockpit builders, setups...).

Reply:
Yes, and the internal impact is also high as unfortunately the rust code adds this prefix automatically, JS doesn't - so it would even be possible to easily change it.

Reply:
A persistent Lvar is quite dangerous. If anything happens to alter it at runtime e.g. replay or shared cockpit tool, then the new value will be stored and override the default defined in systems.cfg for all future sessions. Also note you'd need to use LocalVar.2 if you did add another one.

Reply:
Also, I'd recommend not using 0 for any aircraft. That way you can be sure the Lvar was actually set before/when you use it.

Reply:
Yes - corrected the suggestion. 

But I do not think an LVar is risky. If this would be the case basically no LVar in the aircraft would make sense as they all can be messed with. 
And if someone or a third party changes them then it breaks our aircraft. So what - the borders on intend in this case. 

In any case an LVar would be the easiest solution as it is easy to query from everywhere - even externally whereas a function needs an include and also is not available externally.

Reply:
The difference is you're suggesting a persistent localvar. That means it's state is stored by the sim, and persisted across sessions. The default value (LocalVarDefault.2) is only used if there's no stored state.

Reply:
> The difference is you're suggesting a persistent localvar. That means it's state is stored by the sim, and persisted across sessions. The default value (LocalVarDefault.2) is only used if there's no stored state.

Ah ok - I actually didn't know that - I would have expected this to be read every start.